KenticoCloud.Delivery/InlineContentItems/InlineContentItemsProcessor.cs

```diff
@@ -12,29 +13,29 @@ namespace KenticoCloud.Delivery.InlineContentItems
     internal class InlineContentItemsProcessor : IInlineContentItemsProcessor
     {
         private readonly IDictionary<Type, Func<object, string>> _inlineContentItemsResolvers;
-        private readonly IInlineContentItemsResolver<UnretrievedContentItem> _unretrievedInlineContentItemsResolver;
         private readonly HtmlParser _htmlParser;
         private readonly HtmlParser _strictHtmlParser;
 
-        /// <summary>
-        /// Resolver used in case no other resolver was registered for type of inline content item
-        /// </summary>
-        public IInlineContentItemsResolver<object> DefaultResolver { get; set; }
-
         // Used by tests only
-        internal IEnumerable<Type> ContentItemTypesWithResolver => _inlineContentItemsResolvers.Keys;
+        internal IEnumerable<Type> ContentItemTypesWithResolver => _inlineContentItemsResolvers.Keys.ToArray();
```

Comment:
I don't like "for tests only" :)
Why not to wrap the private field in an internal property and leave it that way without additional comments?

Reply:
Well, it already is an internal property 😉. But it should not be used within the Delivery project itself as it was only meant for tests. If you think the comment is redundant and property won't be misused, let's remove the comment.

Reply:
yeah, i can see that, thank you very much :D 
what I meant is - why not to convert it into an automatic property and do all the `.Keys` and `.ToArray()`s calls outside, in the tests to be precise... now it really feels like a single-purpose solution.

Reply:
I still miss something here, I guess - it is an automatic property, isn't it? 😟 

The `.ToArray` should be here to prevent manipulation with the inner component state. But if you feel like making the `_inlineContentItemsResolvers` into an internal automatic property (is this what you meant [all along](https://memegenerator.net/img/instances/81112171.jpg)? 😊), then go for it. 

I would still prefer a backing field with an internal property that would make a [read-only copy](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.readonlydictionary-2?view=netframework-4.7.2) before providing the collection outside of the `Processor`. Something like:
```suggestion
        internal IReadOnlyDictionary<Type> ContentItemResolvers => new ReadOnlyDictionary<Type, Func<object, string>>(_inlineContentItemsResolvers);
```
But I will leave this completely upon you.

Reply:
see, I was wonder why did you convert the collection to an array. I'll convert it to a readonly collection. this way the intentions will be much clearer :)